src/mono/mono/metadata/appdomain.h

```diff
@@ -94,9 +94,6 @@ mono_domain_unload (MonoDomain *domain);
 MONO_API void
 mono_domain_try_unload (MonoDomain *domain, MonoObject **exc);
 
-MONO_API mono_bool
-mono_domain_is_unloading   (MonoDomain *domain);
```

Comment:
This is a public header.  Need to keep a `MONO_RT_EXTERNAL_ONLY` symbol that always returns false